screens/DetailedMuseumScreen.tsx

```diff
@@ -4,45 +4,116 @@ import {
   StyleSheet,
   Dimensions,
   Image,
+  Text,
+  Animated,
 } from 'react-native';
 import type {NativeStackScreenProps} from '@react-navigation/native-stack';
+import {Chip} from '@rneui/themed';
+import {useState} from 'react';
+import {startCase} from 'lodash';
 
 import {MOCK_IMAGE} from '../utils/constans';
 import type {MainRootStackParamList} from '../App';
 import {getAppStyles} from '../utils/styles';
 
+import ScrollView = Animated.ScrollView;
+
+import {getAppColors} from '../utils/colors';
+
 type Props = NativeStackScreenProps<MainRootStackParamList, 'DetailedMuseum'>;
 
 export default function DetailedMuseumScreen({route}: Props) {
   const colorScheme = useColorScheme();
+  const [selectedField, setSelectedField] = useState<string>(
+    fieldsToDisplay[0],
+  );
 
   const museum = route.params.museum;
   const imageUrl =
     museum.primaryImageSmall.trim().length === 0
       ? MOCK_IMAGE
       : museum.primaryImageSmall;
 
-  const {backgroundStyle} = getAppStyles(colorScheme);
+  const {backgroundStyle, textStyle} = getAppStyles(colorScheme);
+  const {textColor, textInverseColor} = getAppColors(colorScheme);
+
+  const renderChip = (item: string) => {
+    return (
+      <Chip
+        title={startCase(item)}
+        type={selectedField === item ? 'solid' : 'outline'}
+        onPress={() => setSelectedField(item)}
+        buttonStyle={{borderColor: textColor}}
+        color={textColor}
+        titleStyle={{
+          color: item === selectedField ? textInverseColor : textColor,
+        }}
+        style={[styles.chip]}
+      />
+    );
+  };
 
   return (
     <View style={[styles.containerLayout, backgroundStyle]}>
       <Image source={{uri: imageUrl}} style={styles.image} resizeMode="cover" />
+      <View style={styles.chipListWrapper}>
+        <ScrollView
+          style={styles.chipList}
+          horizontal={true}
+          showsHorizontalScrollIndicator={false}>
+          {fieldsToDisplay.map(renderChip)}
+        </ScrollView>
+      </View>
+
+      <Text style={[textStyle, styles.title]}>{startCase(selectedField)}</Text>
+      <Text style={[textStyle, styles.description]}>
+        {(museum[selectedField]?.length ?? 0) === 0
+          ? 'No data available'
+          : museum[selectedField]}
```

Comment:
I think this would be enough, and a bit more readible 😄 
```suggestion
        {museum[selectedField]?.length
          ? museum[selectedField]
          : 'No data available'}
```

Reply:
yea, in my head null / undefined cannot be treated as boolean, but js allows it